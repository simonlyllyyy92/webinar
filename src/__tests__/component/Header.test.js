// import ReactShallowRenderer from "react-test-renderer/shallow"
import React from "react"
import Header from "../../components/header/headerIndex"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

//react-test-renderer
test("should render Header correctly", () => {
  const wrapper = shallow(<Header />)
  expect(toJson(wrapper)).toMatchSnapshot()
  // const renderer = new ReactShallowRenderer()
  // renderer.render(<Header />)
  // expect(renderer.getRenderOutput()).toMatchSnapshot()
})
