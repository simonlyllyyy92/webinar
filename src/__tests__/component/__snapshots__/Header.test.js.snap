// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="headerContainer"
>
  <withRouter(Connect(Toolbar))
    handleOnclick={[Function]}
  />
  <SlideDrawer
    onShow={false}
  />
</div>
`;
